frontend/awx/administration/execution-environments/ExecutionEnvironments.tsx

```diff
@@ -98,7 +101,7 @@ export function ExecutionEnvironments() {
     <PageLayout>
       <PageHeader
         title={t('Execution Environments')}
-        titleDocLink="https://docs.ansible.com/automation-controller/latest/html/userguide/execution_environments.html"
+        titleDocLink={`${getDocsBaseUrl(config)}/html/userguide/execution_environments.html`}
```

Comment:
The page header popover will not render without passing the `titleHelp` prop, so we won't see the link out to the docs. We don't have to fix this as part of this PR, however.

Reply:
Thanks! Updated the titleHelp so that the popover appears and we can see the link to the docs.